exec/java-exec/src/main/java/org/apache/drill/exec/expr/SizedJBlock.java

```diff
@@ -32,7 +32,11 @@
 
   public SizedJBlock(JBlock block) {
     this.block = block;
-    this.count = 0;
+    // count should be 1 because in some cases it is hard to increase it when
+    // Logical Expressions are added to JBlock.
+    // To avoid taking into account of some extra count from empty JBlock,
```

Comment:
I spent some time to understand why modifying this count makes the query compilation works.  

The failing case encounter compilation failure in Copier (for SVR operator). Turns out that the idea of SizedJBlock (DRILL-4715) only works when we call ClassGenerator.addExpr(). This is fine with Project, Filter, Aggregator, etc, but not for Copier.  The Copier is doing kind of short-cut handling, by accessing the eval() and setup() directly [1].

Ideally, we probably should try to see if we can convert Copier using same mechanism in Project/Filter. After some thoughts, I realized doing that might add additional overhead, as the current way is doing the copying directly. 

Given that, I'm fine with this proposed change. Please add comments to explain why we set count = 1. 

[1] https://github.com/apache/drill/blob/master/exec/java-exec/src/main/java/org/apache/drill/exec/vector/CopyUtil.java#L45-L50